docs/change-log.md

```diff
@@ -2,6 +2,23 @@
 
 This document provides an overview of the changes we are making over time. Feel free to reach out to us in case of any specific questions.
 
+
+## Product Update February 24th, 2023
+
+- **New:** Extended [MariaDB](https://docs.humanitec.com/integrations/resource-types/mariadb) and [MySQL](https://docs.humanitec.com/integrations/resource-types/mysql) driver inputs with optional parameter `privilege list` that will be applied to the generated user.
+
+- **Fixed:** It is now possible to use browser's native Back button to return to the [App Overview screen](https://docs.humanitec.com/reference/user-interface/app-overview-screen).
+
+- **Fixed:** Trailing whitespace characters are no longer removed from [Shared App Values and Secrets](https://docs.humanitec.com/using-humanitec/work-with-apps/define-app-values-and-secrets#app-level) inputs.
+
+- **Fixed:** [Creating a delta via the API](https://api-docs.humanitec.com/#tag/Delta) and navigating to it by updating the draft id in the URL no longer causes the UI to be unresponsive.
+
+- **Fixed:** Long deployment comments no longer overflow on the [deployment diff](https://docs.humanitec.com/using-humanitec/work-with-environments/deploy-to-environments/diff-deployments) dialog box. 
+
+- **Fixed:** If a deployment is in progress, the following deployments will no longer be accepted. Instead, they will be rejected with an HTTP 409 error.
+
+- **Improved:** Changed [cloning of the whole environment via UI](https://docs.humanitec.com/using-humanitec/work-with-environments/create-environment#create-a-new-environment) to use delta to match the process for [cloning individual workloads](https://docs.humanitec.com/using-humanitec/work-with-environments/clone-between-environments).
```

Comment:
@sujaya-sys based on the slack discussion added this, let me know what you think!

Reply:
@MartaD I suggest:

- **Changed**: The clone of a whole environment now matches the process for cloning individual workloads. In both cases a new delta is created in the target environment.
- I'd link to https://docs.humanitec.com/using-humanitec/work-with-environments/clone-between-environments once and remove the link to https://docs.humanitec.com/using-humanitec/work-with-environments/create-environment#create-a-new-environment 

That being said, I actually don't think the change has any impact on the user's experience. I understood that this is a purely implementational change. @chrishumanitec am I missing something?